test/update_test.go

```diff
@@ -2,14 +2,15 @@ package test
 
 import (
 	"os"
+	"path/filepath"
 
 	"github.com/stretchr/testify/require"
 )
 
 func (s *CLITestSuite) TestUpdate() {
 	s.T().Skip("Skipping this test until its less flaky")
 
-	configFile := os.Getenv("HOME") + "/.confluent/config.json"
+	configFile := filepath.Join(os.Getenv("HOME"), "/.confluent/config.json")
```

Comment:
The point of using `filepath.Join()` is not to use slashes, so it works on Windows too.
```suggestion
	configFile := filepath.Join(os.Getenv("HOME"), ".confluent", "config.json")
```